### PR TITLE
Improve Georgian localization

### PR DESCRIPTION
**Pull request by akalongman**
Improve Georgian (ka & ka_GE) localization

- [x] Add valid values on Georgian for each test case
- [x] Update library according to tests

Related issue: #1716

### CODE REVIEW

**Comment by kylekatarnls** [comment]
I checked in Google Translation (translate Gregorian to English) and all seem OK. I will implement the changes.

**Comment by kylekatarnls** [comment]
As per moment.js, ka would use 12-hours cloak with AM/PM:
https://github.com/moment/moment/blob/develop/src/locale/ka.js#L21

@akalongman Is it a mistake as you used 24-hours in unit tests?

**Comment by kylekatarnls** [comment]
glibc does not use neither AM/PM

**Comment by akalongman** [comment]
Yes, moment.js uses 12  hours clock, but we have not AM/PM suffixes (we use "6 o'clock of evening" etc.) and for simplify decided to use 24 hours clock. 
Btw Russia also uses 12 hours clock but in the library used 24 hours clock, so I think its ok.
Would be great if we had an option for specifying which format to use

**Comment by kylekatarnls** [comment]
You can still use any format inside isoFormat(), LT, LL, etc. are just shortcuts to most common formats of each language. We should not add options, it would be unmaintainable for the hundred of languages we support.

File: tests/Localization/KaGeTest.php [hunk 1]
```diff
@@ -97,130 +97,130 @@ class KaGeTest extends LocalizationTestCase
         // Carbon::parse('2018-04-10 00:00:00')->isoFormat('DDDo')
         'მე-100',
         // Carbon::parse('2018-02-10 00:00:00', 'Europe/Paris')->isoFormat('h:mm a z')
-        '12:00 დილის cet',
+        'ღამის 12:00 cet',
```
Comment (kylekatarnls) [inline tests/Localization/KaGeTest.php]: This one will not be possible as the format is explicitly `h:mm a z`, order can be changed in macro-formats (LTS, LL, etc.) but not when the order is given explicitly by `isoFormat`

Comment (kylekatarnls) [reply tests/Localization/KaGeTest.php]: About ღამის, დილის, can you give me the hours range for each?

**Comment by akalongman** [comment]
@kylekatarnls so, what fo you think, leave 24 hours clock or not? For us - Georgians its ok, because we use 24 hours format in everyday life.

File: tests/Localization/KaGeTest.php (hunk 1 above)
Comment (akalongman) [reply tests/Localization/KaGeTest.php]: I seen that example in Ru test. Is it mistake as well?

Comment (akalongman) [reply tests/Localization/KaGeTest.php]: ღამის 22:00 - 04:00
დილის 04:00 - 11:00
შუადღის/დღის 11:00 - 16:00
საღამოს 16:00 - 22:00

I'll try to find some official docs about these time ranges

**Comment by kylekatarnls** [comment]
24-hours is OK for LT, LTS, LLL and LLLL formats but, we still need to implement AM/PM equivalent for users that will explicitly call `->isoFormat('h:mm a z')` is such case, we give the user what he ask whatever the language.

**Comment by kylekatarnls** [comment]
I added English matching sentences in unit tests please review them. I will wait a few days so you can check it, then I'll release it in 2.19.1.

**Comment by akalongman** [comment]
I'll take a look in next few days

**Comment by kylekatarnls** [comment]
I still have a problem about weekdays (broken tests), when to use `კვირა` and when to use `კვირას`?

**Comment by akalongman** [comment]
@kylekatarnls 
კვირა is a Sunday (Week as well depending on context), 
კვირას is a On Sunday, when you speak about some action on Sunday, e.g. "I am going abroad on Sunday"
Do you need more examples?

**Comment by kylekatarnls** [comment]
I think I get it. I added the following rule: asking for a day alone კვირა, a day in sentence/date : კვირას.

**Comment by kylekatarnls** [comment]
I merge this one for 2.19.1. It's as good as I can trying to reach your proposition keeping the whole thing working in an automated way. I replaced RegExp with explicit replacements in closures in ka.php so maybe you can check it more easily and propose a PR if you see a mistake.

**Pull request merged**